people.md

```diff
@@ -0,0 +1,47 @@
+# Steering Council and Institutional Partners
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
+- Brian Granger
+- Damian Avila
+- Fernando Perez
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```

Comment:
It's not called that in the governance document.  How about we make a separate pull request to change the name of the committee in both documents?